src/common/lib/theme/ThemeSettingsEditor.tsx

```diff
@@ -60,20 +61,34 @@ export function ThemeSettingsEditor({
     document.documentElement.style.setProperty(key, value);
   }
 
-  const { background, font, fontColor, backgroundHTML, musicURL } =
-    theme.properties;
+  const {
+    background,
+    font,
+    fontColor,
+    headingsFont,
+    headingsFontColor,
+    backgroundHTML,
+    musicURL,
+  } = theme.properties;
 
   useEffect(() => {
     setCSSVar("--user-theme-background", background);
   }, [background]);
 
   useEffect(() => {
-    setCSSVar("--user-theme-font", font);
-  }, [font]);
+    const fontFamily =
+      FONT_FAMILY_OPTIONS_BY_NAME[font]?.config?.style.fontFamily;
+    const headingsFontFamily =
+      FONT_FAMILY_OPTIONS_BY_NAME[headingsFont]?.config?.style.fontFamily;
```

Comment:
We need a compiled font family name (nextjs does it) instead of real font family. Eg `Inter` is `'__Inter_aaf875', '__Inter_Fallback_aaf875'`